src/main/java/uk/ac/cam/cl/dtg/isaac/api/IsaacController.java

```diff
@@ -32,6 +32,7 @@
 import static uk.ac.cam.cl.dtg.segue.api.Constants.SegueServerLogType;
 import static uk.ac.cam.cl.dtg.segue.api.Constants.TYPE_FIELDNAME;
 import static uk.ac.cam.cl.dtg.segue.api.Constants.USER_ID_FKEY_FIELDNAME;
+import static uk.ac.cam.cl.dtg.util.LogUtils.sanitiseExternalLogValue;
```

Comment:
This is an unused import, so I would have expected checkstyle to complain about it.

In any case, I would actually use this, rather than the long form `LogUtils.sanitiseExternalLogValue`, as it would make the lines shorter and easier to read.

Reply:
Huh, that's odd. I normally do use the direct import so I'm not sure where all the long-form usages came from...
Hmmm... looks like (a lot of?) the instances come from where I was replacing the old function reference... maybe something to do with adding the import from the finder window?
Regardless, I'll see about getting them changed over...